instrumentation/spring/spring-boot-autoconfigure/README.md

```diff
@@ -392,6 +440,13 @@ If an exporter is present in the classpath during runtime and a spring bean of t
 
 <!-- Slf4j Log Correlation  otel.springboot.loggers.slf4j.enabled		true   		org.slf4j.MDC -->
 
+##### Resource Properties
+
+| Feature  | Property                                         | Default Value        | ConditionalOnClass |
+|----------|--------------------------------------------------|----------------------|--------------------|
+| Resource | otel.springboot.resource.enabled                 | false                | OsResource         |
+|          | otel.springboot.resource.attributes.service.name | unknown_service:java |                    |
```

Comment:
i'm unsure of the default value, it's sort of `spring.application.name`?

Reply:
no, it is a default name of resource taken from https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/common/src/main/java/io/opentelemetry/sdk/resources/Resource.java#L48
So Resource is created with this name when `spring.application.name` and `otel.springboot.resource.attributes.service.name` are not set